### PR TITLE
Create post route

### PR DESCRIPTION
**Pull request by MaxHalleran**
# Description

Adds a Post Post route, allowing users to create new posts.

### CODE REVIEW

File: MyFirstBlog/Controllers/PostsController.cs [hunk 1]
```diff
@@ -33,4 +34,20 @@ public class PostsController : ControllerBase {
...
     }
+
+    // POST: api/Posts
+    [HttpPost]
+    public ActionResult<PostDto> AddPost(Post post)
+    {
+        try
+        {
+            _postService.AddPost(post);
+            return Ok(post.AsDto());
```
Comment (Nolan-E) [inline MyFirstBlog/Controllers/PostsController.cs:45]: Does this return a 201 on successful create?

**Review by Nolan-E — COMMENTED** [review]
Just a quick clarification

File: MyFirstBlog/Controllers/PostsController.cs (hunk 1 above)
Comment (MaxHalleran) [reply MyFirstBlog/Controllers/PostsController.cs:45]: This returns a 200

Comment (MaxHalleran) [reply MyFirstBlog/Controllers/PostsController.cs]: Changed to return a 201 response.

**Review by Nolan-E — APPROVED** [review]
🌮

**Pull request merged**